lib/src/globals.ts

```diff
@@ -8,16 +8,18 @@ const cpuCount = Platform.isMobileApp ? 1 : require('os').cpus().length
 
 const ocrBackgroundProcesses = cpuCount == 2 ? 1 : 2
 
+const officeBackgroundProcesses = 1
```

Comment:
You can leave it at 1.

It's a configurable value because initially I tried to balance the number of background workers between the OCR, PDF, and total number of available threads. The goal was to maximize the use of resources to extract files as quickly as possible.

It should probably be refactored, but in the meantime, `1` will be good enough.